src/ipv4.c

```diff
@@ -410,11 +410,15 @@ static int ipv4_rcv_fin(struct rte_mbuf *mbuf)
         }
         return ipv4_forward(mbuf);
     } else {
-        RTE_LOG(DEBUG, IPV4, "%s: input route has no dst\n", __func__);
-        route4_put(rt);
-        return EDPVS_KNICONTINUE; /* KNI may like it, don't drop */
+	RTE_LOG(DEBUG, IPV4, "%s: input route has no dst\n", __func__);
+        if (etype != ETH_PKT_HOST) {
+            /* multicast or broadcast */
+            route4_put(rt);
+            return EDPVS_KNICONTINUE; /* KNI may like it, don't drop */
+        }
```

Comment:
Ｉｆ pkt is "no route", means the dest is not for our own IP or KNI, and no idea how to "forward" it to other network. So why re-send the pkt "no route"?